slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/SignedAttestationsDao.java

```diff
@@ -153,4 +153,20 @@ public void insertAttestation(final Handle handle, final SignedAttestation signe
         .mapTo(UInt64.class)
         .findFirst();
   }
+
+  public Optional<UInt64> maximumTargetEpoch(final Handle handle, final int validatorId) {
```

Comment:
The minimumSourceEpoch and minimumTargetEpoch functions can be now be removed as they shouldn't be in used anymore

Reply:
done